elephant/spike_train_generation.py

```diff
@@ -186,21 +186,34 @@ def spike_extraction(
         Default: None
     interval : tuple of :class:`pq.Quantity`
         Specifies the time interval around the `time_stamps` where the waveform
-        is extracted.
+        is extracted. The default values [-2ms, 4ms] are based on experience,
```

Comment:
If this is a tuple, it is better to use `(`, `)` instead of square brackets to be consistent

Reply:
However, if the brackets are referring to the mathematical interval notation to the edges being inclusive, then it is clearer if "default values" is changed to the "default time interval"

Reply:
I would favor the latter as in line 216, the values between brackets refer to the word interval.